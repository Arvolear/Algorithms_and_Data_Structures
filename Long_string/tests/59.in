1405546065739749278329774469728080565363
560529790239906565216296873346213465357