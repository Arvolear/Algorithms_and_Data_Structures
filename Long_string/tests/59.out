1966075855979655843546071343074294030720
845016275499842713113477596381867100006
787850441401627586519742686952946661508557168197803418871914875925407974629591
2